.github/workflows/autobuild.yaml

```diff
@@ -143,6 +143,9 @@ jobs:
         asset_name: clangd-${{ matrix.config.name }}-${{ github.event.release.tag_name }}.zip
         asset_path: clangd.zip
         asset_content_type: application/zip
+    - name: Check binary compatibility
+      if: matrix.config.name == 'linux'
+      run: .github/worflows/lib_compat_test.py --lib=GLIBC_2.18
```

Comment:
maybe mention the reasoning for 2.18, (thread local destructors ?)

Reply:
Good point, Added to /releases.md